data-structures-stacks/get-2nd-from-top.js

```diff
@@ -0,0 +1,5 @@
+export default function get2ndFromTop(stack) {
+  if (stack.peek() === undefined) {
+    return undefined;
+  }
+}
```

Comment:
What about if the stack isn't empty? So far it isn't really getting a value if there is one to get.